src/block-ciphers.rst

```diff
@@ -12,109 +12,103 @@ Block ciphers
 by that digit. :numref:`fig-BlockCipherBlocks`
-illustrates the blocks that the cipher operates on.
+illustrates blocks that the cipher operates on.
 
 
 .. _fig-BlockCipherBlocks:
 
 .. figure:: Illustrations/BlockCipher/AllNodes.svg
    :align: center
 
-   All of the 16 nodes operated on by the block cipher. Each node is designated by a hexadecimal digit.
+   All 16 nodes operated on by the block cipher. Each node is designated by a hexadecimal digit.
 
-Once we select a secret key, the block cipher will use that to determine
-what the encryption of any given block is. We will illustrate that
-relationship with an arrow: the block at the start of the arrow,
-encrypted using :math:`E` under key :math:`k`, is mapped to the block at
-the end of the arrow.
+Once we select a secret key, the block cipher uses it to determine
+the encryption of any given block. We illustrate that
+relationship with an arrow. The tail of the arrow has the block
+encrypted with :math:`E` under key :math:`k` and the arrowhead is mapped to the block.
 
 .. _fig-BlockCipherEncryption:
 
 .. figure:: Illustrations/BlockCipher/Encryption.svg
    :align: center
 
-   An encryption permutation produced by the block cipher under a particular key :math:`k`.
+   An encryption permutation made by a block cipher under a particular key :math:`k`.
 
-In :numref:`fig-BlockCipherEncryption`, you'll notice
-that the permutation isn't just one big cycle: there's a large cycle of
-7 elements, and several smaller cycles of 4, 3 and 2 elements each. It's
+In :numref:`fig-BlockCipherEncryption`, note
+that the permutation is not just one big cycle. It contains a large cycle of
+7 elements, and several smaller cycles of 4, 3 and 2 elements each. It is
 also perfectly possible that an element encrypts to itself. This is to
 be expected when selecting random permutations, which is approximately
 what a block cipher is doing; it doesn't demonstrate a bug in the block
 cipher.
 
-When you're decrypting instead of encrypting, the block cipher just
+When you decrypt instead of encrypt, the block cipher 
 computes the inverse permutation. In :numref:`fig-BlockCipherDecryption`,
-you can see that we get the same illustration, except that all the arrows are
-going in the other direction.
+we get the same illustration. The exception is that all arrowheads point
+in the opposite direction.
 
 .. _fig-BlockCipherDecryption:
 
 .. figure:: Illustrations/BlockCipher/Decryption.svg
    :align: center
 
    The decryption permutation produced by the block cipher under the same key
-   :math:`k`: the inverse of the encryption permutation, that is: all the arrows
-   have been reversed.
+   :math:`k`. It is the inverse of the encryption permutation in that all arrowheads
+   reverse.
 
 
-The only way to know which block maps to which other block, is to know
-the key. A different key will lead to a completely different set of
+Knowing the key helps in understanding which block maps to other blocks.
```

Comment:
It's important that the key defines which blocks map to which; perhaps "helps in understanding" implies that you know e.g. 10% of them normally but you need the key to know all of them, whereas in reality you need the key to know _any_ of them. Consider "The key defines which blocks map to which blocks". (It's also maybe not entirely accurate to say "which blocks map to which other blocks" since you'll typically have a few blocks that map to themselves under any key. They're rare, but it happens.)